### DISCUSSION TITLE
Improve CLOMonitor Score + consider a public meeting to help grow community

### DISCUSSION THREAD

**Discussion by caniszczyk** [keycloak/keycloak#30843] [Ideas]
### Description

The Keycloak community can consider improving its https://clomonitor.io/projects/cncf/keycloak score :)

Also I see that there is no public community meeting which isn't a best practice.

https://clomonitor.io/docs/topics/checks/#community-meeting

<img width="1104" alt="Screenshot 2024-06-26 at 11 11 12 AM" src="https://github.com/keycloak/keycloak/assets/63777/db94912c-12d7-4db2-bd79-6bd3cc9972be">

I'd suggest a monthly meeting like https://github.com/prometheus-community/community/blob/master/MEETING.md or https://github.com/kubernetes/community/blob/master/events/community-meeting.md or

**Comment by y-tabata** [comment]
@caniszczyk 
Currently, with my PR (https://github.com/keycloak/keycloak/issues/32384), the Community meeting description has been added to the README and the CLOMonitor score has been increased.
![image](https://github.com/user-attachments/assets/2560f72d-2e3b-4d8a-a5fe-bfd407506017)

**Comment by y-tabata** [comment]
@caniszczyk 
My PR (#36127) added an OpenSSF Scorecard badge to the README and increased the CLOMonitor score.
![image](https://github.com/user-attachments/assets/0031196c-6f2c-4181-a46a-13ce7b53b4d7)
